evalml/tests/automl_tests/test_automl.py

```diff
@@ -1795,7 +1800,7 @@ def test_automl_error_callback(mock_fit, mock_score, X_y_binary, caplog):
         assert str(e) == msg
 
     caplog.clear()
-    automl = AutoMLSearch(problem_type="binary", error_callback=raise_and_save_error_callback)
+    automl = AutoMLSearch(problem_type="binary", error_callback=raise_and_save_error_callback, train_best_pipeline=False)
```

Comment:
Same question here as in `test_automl_error_callback`

Reply:
Similar reason as above